### PR TITLE
dump all isas with amdllpc

### PR DESCRIPTION
**Pull request by aldesilv**
cd ~/SHARK/shark/examples/shark_inference/stable_diffusion
python main.py --precision="fp16" --device="vulkan" --iree-vulkan-target-triple=rdna3-unknown-linux --no-load_vmfb --dispatch_benchmarks="all" --dispatch_benchmarks_dir="SD_dispatches" --dump_isa

### CODE REVIEW

File: shark/iree_utils/compile_utils.py [hunk 1]
```diff
@@ -90,6 +90,17 @@ def create_dispatch_dirs(bench_dir, device):
                 )
 
 
+def dump_isas(bench_dir):
+  for d_ in os.listdir(bench_dir):
+      if os.path.isdir(f"{bench_dir}/{d_}"):
+          for f_ in os.listdir(f"{bench_dir}/{d_}"):
+              if f_.endswith('.spv'):
+                  os.system(
+                      f"/usr/bin/amdllpc -gfxip 11.0 {bench_dir}/{d_}/{f_} -v > \
```
Comment (powderluv) [inline shark/iree_utils/compile_utils.py:99]: can we just call `amdllpc` so it can be anywhere in your path not just /usr/bin ?

**Comment by powderluv** [comment]
Please add an example usage in the commit message. and fix up lint

File: shark/iree_utils/compile_utils.py (hunk 1 above)
Comment (aldesilv) [reply shark/iree_utils/compile_utils.py:99]: amdllpc: command not found, even though /usr/bin is in $PATH. also I have to use sudo /usr/bin/amdllpc ... or else Permission denied

Comment (powderluv) [reply shark/iree_utils/compile_utils.py]: Wait I may need to set perms on amdllpc. You shouldn't have to use sudo

**Review by powderluv — APPROVED** [review]

**Pull request merged**